packages/design-tokens/stories/Shadows.mdx

```diff
@@ -13,39 +13,208 @@ Shadows come in four different sizes: XS, SM, MD, and LG. The size of the shadow
 
 <Canvas of={ShadowStories.Size} />
 
-| Size   | JS                | CSS                     |
-| ------ | ----------------- | ----------------------- |
-| **XS** | `shadows.size.xs` | `var(--shadow-size-xs)` |
-| **SM** | `shadows.size.sm` | `var(--shadow-size-sm)` |
-| **MD** | `shadows.size.md` | `var(--shadow-size-md)` |
-| **LG** | `shadows.size.lg` | `var(--shadow-size-lg)` |
```

Comment:
Unfortunately markdown tables have stopped working in storybook so we have to use jsx

<img width="1050" alt="Screenshot 2024-11-22 at 11 57 14 AM" src="https://github.com/user-attachments/assets/56a2fce4-3041-4781-8b31-a3e0a711e655">